sample-eksctl-ssh.yaml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Nice!

I think this probably should live under something like a `docs/` prefix. This will otherwise be yet another set of files right at the top :disappointed:

Reply:
I agree that we could think about a file reorganization, but given that we expect anyone trying Bottlerocket to use one of these, I thought they were important and it was reasonable (for now) for them to be this easy to find.

Reply:
SGTM, I'm sure we'll see and revisit this based on where it's going - ship it!